protocol/mayan/api.go

```diff
@@ -41,7 +41,7 @@ func NewMayanExplorer() *MayanExplorer {
 }
 
 func (c *MayanExplorer) GetSwap(hash string) (*MayanSwap, error) {
-	url := fmt.Sprintf("%s/v3/swap/order-id/SWIFT_%s", MAYAN_EXPLORER_URL, hash)
+	url := fmt.Sprintf("%s/v3/swap/order-id/%s", MAYAN_EXPLORER_URL, hash)
```

Comment:
not sure this PR is the right place for this change

Reply:
This is a mistake :fearful: